README.md

```diff
@@ -1 +1,5 @@
-# tribute-page
```

Comment:
This page should be more described by you. This file represents your professionalism in coding.  You can follow [Microverse Template](https://github.com/microverseinc/readme-template) for more info